pkg/agent/option.go

```diff
@@ -95,6 +98,7 @@ func (opts *ClusterRegistrationOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&opts.ClusterStatusCollectFrequency.Duration, ClusterStatusCollectFrequency, opts.ClusterStatusCollectFrequency.Duration,
 		"Specifies how often the agent collects current child cluster status")
 	fs.BoolVar(&opts.TunnelLogging, "enable-tunnel-logging", opts.TunnelLogging, "Enable tunnel logging")
+	fs.StringVar(&opts.PredictorAddress, "external-predictor-addr", opts.PredictorAddress, "set external predictor address, if not set , use default predictor(in agent)")
```

Comment:
The address of external predictor.  If not set, the default built-in predictor will be used.